src/main/java/io/github/emilyydev/bypass_resource_pack/mixin/ServerPackStatusMixin.java

```diff
@@ -0,0 +1,74 @@
+//
+// Bypass Resource Pack - Fabric mod to reject and bypass forced resource packs
+// Copyright (C) 2022  emilyy-dev
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Lesser General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Lesser General Public License for more details.
+//
+// You should have received a copy of the GNU Lesser General Public License
+// along with this program.  If not, see <https://www.gnu.org/licenses/>.
+//
+
+package io.github.emilyydev.bypass_resource_pack.mixin;
+
+import net.minecraft.client.multiplayer.ServerData;
+import net.minecraft.network.chat.Component;
+import org.spongepowered.asm.mixin.Final;
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.Mutable;
+import org.spongepowered.asm.mixin.Shadow;
+import org.spongepowered.asm.mixin.Unique;
+import org.spongepowered.asm.mixin.gen.Invoker;
+import org.spongepowered.asm.mixin.injection.At;
+import org.spongepowered.asm.mixin.injection.Inject;
+import org.spongepowered.asm.mixin.injection.callback.CallbackInfoReturnable;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+import static io.github.emilyydev.bypass_resource_pack.BypassableConfirmScreen.BYPASS_TEXT;
+
+@Mixin(ServerData.ServerPackStatus.class)
+@Unique
+public abstract class ServerPackStatusMixin {
+
+  @Shadow
+  @Final
+  @Mutable
+  private static ServerData.ServerPackStatus[] $VALUES;
```

Comment:
```suggestion
  @Shadow @Final @Mutable private static ServerData.ServerPackStatus[] $VALUES;
```
While you're at it :d